pom.xml

```diff
@@ -32,12 +32,13 @@
         <dependency>
             <groupId>net.dv8tion</groupId>
             <artifactId>JDA</artifactId>
-            <version>4.4.1_353</version>
+            <version>5.0.0-beta.15</version>
         </dependency>
+        <!-- Must be modified when the PullRequest will be accepted. -->
         <dependency>
-            <groupId>com.jagrosh</groupId>
+            <groupId>com.github.DxsSucuk</groupId>
             <artifactId>jda-utilities</artifactId>
-            <version>3.0.5</version>
+            <version>fdbc298</version>
             <type>pom</type>
         </dependency>
```

Comment:
Revert to the upstream project if possible. I'm guessing since jda-utils is currently not compatible with JDA 5, wait for the upcoming update.